deep-learning/README.md

```diff
@@ -45,7 +45,6 @@ In these courses, you will learn the foundations of Deep Learning, understand ho
 
 1. [CS224n: Natural Language Processing with Deep Learning](http://web.stanford.edu/class/cs224n/)
```

Comment:
It would be great if you could move the remaining two courses for NLP, out of the DL readme and add them to the NLP readme in "Other Courses".

Reply:
@GlazeDonuts it will be better if you can do these changes and close the PR. 
We already have one folder for NLP, maybe add there